engine/dlib/src/cs/dmsdk/dlib/configfile_gen.cs

```diff
@@ -0,0 +1,53 @@
+// Generated, do not edit!
+// Generated with cwd=/Users/mathiaswesterdahl/work/defold/engine/dlib and cmd=/Users/mathiaswesterdahl/work/defold/scripts/dmsdk/gen_sdk.py -i /Users/mathiaswesterdahl/work/defold/engine/dlib/sdk_gen.json
+
+// Copyright 2020-2024 The Defold Foundation
+// Copyright 2014-2020 King
+// Copyright 2009-2014 Ragnar Svensson, Christian Murray
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+// 
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+// 
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+using System.Runtime.InteropServices;
+using System.Reflection.Emit;
+
+
+namespace dmSDK.Dlib {
+    public unsafe partial class ConfigFile
+    {
+        /*#
+        * Generated from [ref:ConfigFile]
```

Comment:
Do we want to generate docs for c# in the future based on our docs?

Reply:
Yes, question is how?
It's the ticket https://github.com/defold/defold/issues/9067